Src/Support/Google.Apis/Responses/HttpResponseMessageExtensions.cs

```diff
@@ -34,9 +38,9 @@ internal static class HttpResponseMessageExtensions
         /// </list>
         /// Any exception thrown while reading the <paramref name="response"/> <see cref="HttpResponseMessage.Content"/>
         /// will be bubbled up.
-        /// Otherwie this method will returned the deserialized <see cref="RequestError"/>.
+        /// Otherwie this method will return the deserialized <see cref="RequestError"/>.
```

Comment:
Nit: Otherwie => Otherwise (existing typo, but let's fix both at the same time)

Reply:
Done!